website/pages/docs/connect/wan-federation-via-mesh-gateways.mdx

```diff
@@ -0,0 +1,185 @@
+---
+layout: docs
+page_title: Connect - WAN Federation via Mesh Gateways
+sidebar_title: WAN Federation via Mesh Gateways <sup> Beta </sup>
+description: |-
+  WAN federation via mesh gateways allows for Consul servers in different datacenters to be federated exclusively through mesh gateways.
+---
+
+# WAN Federation via Mesh Gateways <sup>Beta</sup>
+
+-> **1.8.0+:**  This feature is available in Consul versions 1.8.0 and higher
+
+~> This topic requires familiarity with [mesh gateways](/docs/connect/mesh_gateway).
+
+WAN federation via mesh gateways allows for Consul servers in different datacenters
+to be federated exclusively through mesh gateways.
+
+When setting up a
+[multi-datacenter](https://learn.hashicorp.com/consul/security-networking/datacenters)
+Consul cluster, operators must ensure that all Consul servers in every
+datacenter must be directly connectable over their WAN-advertised network
+address from each other.
+
+If you are using Kubernetes, refer to our [Kubernetes Multi Cluster](/docs/k8s/installation/multi-cluster) documentation.
```

Comment:
@lkysow this is more of a placeholder line for now, feels like the wrong place to say this here, seems like it should be in its own section or maybe the first sentence of this topic. thoughts?

Reply:
I think this is fine here but could also live in an alert at the top of the page.